### PR TITLE
Use `python -m pip` instead of the `pip` executable

### PR DESCRIPTION
**Pull request by mshr-h**
Because it's the preferred invocation method.

[Issue 22295: Clarify available commands for package installation - Python tracker](https://bugs.python.org/issue22295)

### CODE REVIEW

**Comment by ksaur** [comment]
LGTM! @interesaaat can you double check? :)

File: hummingbird/ml/operator_converters/sklearn/gbdt.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 def _get_n_features(model):
     try:
-        return model.n_features_
+        return model.n_features_in_
     except AttributeError:
         # HistGradientBoosting
         return model._n_features
```
Comment (interesaaat) [inline hummingbird/ml/operator_converters/sklearn/gbdt.py]: For HistGradientBoosting should we also use `n_features_in_`?

Comment (mshr-h) [reply hummingbird/ml/operator_converters/sklearn/gbdt.py]: In the official docs, there's `n_features_in_` in `HistGradientBoosting` but not `n_features_`.
I think we should use `n_features_in_` for `HistGradientBoosting`.

[sklearn.ensemble.HistGradientBoostingClassifier — scikit-learn 1.1.1 documentation](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingClassifier.html)
[sklearn.ensemble.HistGradientBoostingRegressor — scikit-learn 1.1.1 documentation](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingRegressor.html)

Comment (interesaaat) [reply hummingbird/ml/operator_converters/sklearn/gbdt.py]: Ok, let's remove the `except` part and use `n_features_in_` everywhere?

Comment (mshr-h) [reply hummingbird/ml/operator_converters/sklearn/gbdt.py]: Sounds good. To keep the PR small, we should work on another branch.
I've opened a new issue #606 to track that.

**Review by interesaaat — APPROVED** [review]

**Pull request merged**